index.html

```diff
@@ -26,24 +26,44 @@
       <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/fontawesome.css" integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous"/>
 			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/regular.min.css" integrity="sha512-k2UAKyvfA7Xd/6FrOv5SG4Qr9h4p2oaeshXF99WO3zIpCsgTJ3YZELDK0gHdlJE5ls+Mbd5HL50b458z3meB/Q==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 		</head>
+    
 		<body>
+
 			<main class="main">
+        <nav class="mobileNavItem">   
+          
+            <ul id="menu" class="menu">
+              <div class="mobileNavItemu">
```

Comment:
Good job on the functionality but you still have problems with the design. Kindly fix your design according to Figma template as the previous reviewer requested. Currently, it still looks like the below.
![Screen Recording 2023-01-06 at 7 47 01 PM](https://user-images.githubusercontent.com/25887253/211059515-d435890f-db7d-49b9-8f8b-190e0edb1a26.gif)

Figma template:
<img width="229" alt="Screenshot 2023-01-06 at 7 47 47 PM" src="https://user-images.githubusercontent.com/25887253/211059637-e41ac68c-2080-4c87-96d2-1e882aa98f32.png">

Reply:
sorry @huseyinbiyik that was not what i updated